src/components/apikey/APIKeyForm.tsx

```
@@ -0,0 +1,348 @@
+        },
+        secretRef: {
+          name: '',
+        },
+        planTier: '',
+        requestedBy: {
+          userId: '',
+          email: '',
+        },
+        useCase: '',
+      },
+    }
```

ESLint

[error] Insert `,`

(prettier/prettier)